CHANGELOG.md

```diff
@@ -22,6 +22,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - The default value for `draw_bold_text_with_bright_colors` is now `false`
 - Mirror OSC query terminators instead of always using BEL
 - Increased Beam, Underline, and Hollow Block cursors' line widths
+- Dynamic title is not disabled anymore when `window.title` is set in config
+- Live config reload for `window.title`
```

Comment:
It shouldn't be in `Changed` I guess, it was in `Added` before IIRC, why did you move it?

Reply:
Oh, it was in `Fixed` and I think I moved it to the wrong section, thanks.